### PR TITLE
HV-1074 Add Arabic translations

### PR DESCRIPTION
**Pull request by flibbertigibbet**

### CODE REVIEW

**Comment by gunnarmorling** [comment]
@flibbertigibbet Awesome, thanks for this!

I saw there are some remaining ones in English, can those not translated for some reason? If so, they could even be removed from this file, the resource bundle mechanism will automatically fall back to the default (English) bundle if a given key is not found.

**Comment by flibbertigibbet** [comment]
Okay, I've pulled out the untranslated strings and rebased. Our translator wasn't sure what to do with the checksum messages or the Brazilian taxpayer messages, so we didn't get anything back for those.

File: engine/src/main/resources/org/hibernate/validator/ValidationMessages_ar.properties [hunk 1]
```diff
@@ -0,0 +1,26 @@
+
+javax.validation.constraints.AssertFalse.message = \u064a\u062c\u0628 \u0623\u0646 \u062a\u0643\u0648\u0646 \u062e\u0627\u0637\u0626\u0629
+javax.validation.constraints.AssertTrue.message  = \u064a\u062c\u0628 \u0623\u0646 \u062a\u0643\u0648\u0646 \u0635\u062d\u064a\u062d\u0629
+javax.validation.constraints.DecimalMax.message  = \u064a\u062c\u0628 \u0623\u0646 \u062a\u0643\u0648\u0646 \u0623\u0642\u0644 \u0645\u0646 ${inclusive == true ? 'or equal to ' : ''}{value}
+javax.validation.constraints.DecimalMin.message  = \u064a\u062c\u0628 \u0623\u0646 \u062a\u0643\u0648\u0646 \u0623\u0643\u0628\u0631 \u0645\u0646 ${inclusive == true ? 'or equal to ' : ''}{value}
+javax.validation.constraints.Digits.message      = \u0627\u0644\u0642\u064a\u0645\u0629 \u0627\u0644\u0631\u0642\u0645\u064a\u0629 \u062e\u0627\u0631\u062c \u0627\u0644\u062d\u062f\u0648\u062f (<{integer} \u0623\u0631\u0642\u0627\u0645>. <{fraction} \u0623\u0631\u0642\u0627\u0645> \u0645\u062a\u0648\u0642\u0639\u0629)
```
Comment (gsmet) [inline engine/src/main/resources/org/hibernate/validator/ValidationMessages_ar.properties]: Hi @flibbertigibbet 

Could you please also translate the 'is equal to' string inside the quotes? It's part of the message depending on if the constraint is inclusive or not. Note that the position of this string might not be appropriate for Arabic so you might need to adjust it.

Thanks!

**Comment by flibbertigibbet** [comment]
Got a translation for 'or equal to' in there and rebased.

**Comment by gsmet** [comment]
Yes, they are very specific. Nice to have the basics covered!

**Comment by gsmet** [comment]
Looks good to me. Could you sign the CLA for Hibernate Validator here: https://cla.jboss.org/ ?

**Comment by flibbertigibbet** [comment]
Okay, agreement signed. Thanks!

**Comment by gunnarmorling** [comment]
Merged; Thanks @flibbertigibbet!

**Pull request merged**